src/main/java/ch/heigvd/dai/commands/Client.java

```diff
@@ -58,9 +60,31 @@ public class Client implements Callable<Integer> {
   private BufferedWriter output;
 
   private AtomicBoolean isDead = new AtomicBoolean(false);
-
+  private int score = 0;
+  private int bestScore = 0;
+
+  /**
+   * The call method is the main method of the client.
+   * It's call by Root when the client command is executed.
+   * @return 0 for success
+   * @throws InterruptedException
+   * @throws UnknownHostException
+   * @throws IOException
+   */
   @Override
   public Integer call() throws InterruptedException, UnknownHostException, IOException {
```

Comment:
Since this method is supposed to return 0 for succes and 1 for error, we can catch all the exception here and return 1 in the catch clause. This would make it possible to remove `throws InterruptedException, UnknownHostException, IOException`.